qiita_db/metadata_template/test/test_util.py

```diff
@@ -246,20 +246,6 @@ def test_get_get_invalid_sample_names_mixed(self):
         obs = qdb.metadata_template.util.get_invalid_sample_names(one_invalid)
         self.assertItemsEqual(obs, [' ', ' ', ' '])
 
-    def test_invalid_lat_long(self):
-
-        with self.assertRaises(qdb.exceptions.QiitaDBColumnError):
-            obs = qdb.metadata_template.util.load_template_to_dataframe(
-                StringIO(SAMPLE_TEMPLATE_INVALID_LATITUDE_COLUMNS))
```

Comment:
By removing these tests, the `SAMPLE_TEMPLATE_INVALID_LATITUDE_COLUMNS` looks like is not used anywhere. Can you remove it?